config/environments/development.rb

```diff
@@ -14,11 +14,29 @@
   # Show full error reports.
   config.consider_all_requests_local = true
 
-  # Enable server timing
+  # Enable server timing.
   config.server_timing = true
 
-  # Enable/disable caching. By default caching is disabled.
-  # Run rails dev:cache to toggle caching.
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000, protocol: 'http' }
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address: 'smtp.gmail.com',
+    port: 587,
+    domain: 'gmail.com',
+    user_name: 'brightcoders321@gmail.com',
+    password: 'fqyqazczsufkdtaq',
```

Comment:
This shouldn't be here 
Use [Rails credentials](https://blog.kiprosh.com/rails-encrypted-credentials/)